jb/itch5/ut_order_book.cpp

```diff
@@ -3,11 +3,12 @@
 #include <boost/test/unit_test.hpp>
 
 /**
- * @test Verify that jb::itch5::order_book works as expected.
+ * @test Verify that jb::itch5::order_book<jb::itch5::map_price> works as
```

Comment:
Please take a look at jb/fftw/ut_plan.cpp in a future PR you might want to refactor the tests to template functions and just call them with different types.